app/src/main/java/org/xbmc/kore/ui/NowPlayingFragment.java

```diff
@@ -311,18 +321,7 @@ public void onVolumeMuteClicked(View v) {
         action.execute(hostManager.getConnection(), new ApiCallback<Boolean>() {
             @Override
             public void onSuccess(Boolean result) {
-                if (!isAdded()) return;
-                if (result) {
-                    Resources.Theme theme = getActivity().getTheme();
-                    TypedArray styledAttributes = theme.obtainStyledAttributes(new int[] {
-                            R.attr.colorAccent});
-                    volumeMuteButton.setColorFilter(
-                            styledAttributes.getColor(0,
-                                    getActivity().getResources().getColor(R.color.accent_default)));
-                    styledAttributes.recycle();
-                } else {
-                    volumeMuteButton.clearColorFilter();
-                }
+                setVolumeMuteButton(!result);
```

Comment:
@SyncedSynapse yup, it is. I use the same solution @tomerf implemented in this PR to set the volume state on resume. But I also use the HostConnectionObserver to set the volume state as soon as it changes. BTW, I did not have to take the complement of the mute state. Strange that @tomerf does need it...
 
@tomerf I'm currently working on showing the volume level on the NowPlayingFragment. It also includes setting the mute state. 
See https://github.com/poisdeux/Kore/tree/feature/%2330
I can rebase my branch on this PR when you fix it.
But I'm not going to wait for it...